tools/timedclient/timedclient.cpp

```diff
@@ -1414,7 +1414,7 @@ static QList<uint> do_search(char *args)
       fprintf(stderr, "missing attribute key or value: %s=%s\n", key, val);
       continue;
     }
-    attributes.insert(key, val);
+    attributes.insert(key, QString(val));
```

Comment:
Suppose it works but these char* conversions can warn or even be unavailable, and at least are vague on the encoding. Guess QString::fromUtf8() could be slightly better here.